.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,110 @@
+---
+name: Feature request
+about: Suggest a new feature for this project
+title: "[NEW FEATURE REQUEST]"
+labels: Feature
+assignees: ''
+
+---
+
+## General information
+
+- Applicant:
+- Developer:
+- User group referent:
```

Comment:
Who is that? Contact person in user group? Do we still need that? Or what would be helpful here?

Reply:
Applicant -> provider
Developer -> developer
Client -> end user